app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -20,6 +26,92 @@ public function __invoke(Request $request, AuditableModel $auditable)
             $auditStatus->entity_name = $auditable->getAuditableNameAttribute();
         }
 
-        return AuditStatusResource::collection($auditStatuses);
+        $transformedAudits = $this->transformAudits($this->getAudits($auditable), $auditable);
+
+        $combinedData = $auditStatuses->concat($transformedAudits);
+
+        return AuditStatusResource::collection($combinedData);
+    }
+
+    private function getAudits($auditable)
+    {
+        $auditsModelInstance = $this->getModelInstance($auditable);
```

Comment:
I don't understand what's going on with this `getModelInstance` method. `$auditable` is already a model instance, and that method appears to be returning a model *class*, not instance, which means the `->audits()` call below doesn't look like it would function at all. Can you enlighten me as to how this works? At the very least, I don't think the `getModelInstance()` method should be needed at all; avoiding methods like that is the whole purpose of the ModelBindingInterfaceMiddleware, which this endpoint is using.